socorro/cron/jobs/bugzilla.py

```diff
@@ -155,10 +145,14 @@ def inner_transaction(
         if not useful:
             raise NothingUsefulHappened('nothing useful done')
 
-    def _iterator(self, query):
-        for report in csv.DictReader(urllib2.urlopen(query)):
+    def _iterator(self, from_date):
+        payload = BUGZILLA_PARAMS.copy()
+        payload['chfieldfrom'] = from_date
+        r = requests.get(BUGZILLA_BASE_URL, params=payload)
+        results = r.json()
```

Comment:
Oo--I missed this because reasons. I agree with this, too. Definitely worth doing something here so that errors are clearer for future debuggers.